File: benchmarks/programs/fibonacci/fibonacci.go
package main

import "fmt"

func main() {
	n := 30
	fmt.Println(0)
	fmt.Println(1)
	a, b := 0, 1
	for i := range n - 2 {
		_ = i
		temp := a + b
		fmt.Println(temp)
		a = b
		b = temp
	}
}

File: benchmarks/programs/fibonacci/fibonacci.rs
fn main() {
    let n = 30;
    println!("0");
    println!("1");
    let (mut a, mut b) = (0_i64, 1_i64);
    for _ in 3..=n {
        let temp = a + b;
        println!("{}", temp);
        a = b;
        b = temp;
    }
}
